lib/ardb/cli/commands.rb

```diff
@@ -197,6 +205,8 @@ def run(argv, *args)
         migration = Ardb::Migration.new(Ardb.config, @clirb.args.first)
         migration.save!
         @stdout.puts "generated #{migration.file_path}"
+      rescue Ardb::InvalidDBFileError
+        raise
```

Comment:
@jcredding so it really sucks to have to repeat this on each cmd (and also repeat tests for each cmd).  What if we move the `Ardb.init` statement out of this rescue?  Then both the invalid db file error and any standard errors will be handled by the generic CLI error handling (which I think has a similar effect).  You'd lose the custom error message but that's it.  I'd argue that any error the init statement should just get the generic handling as it isn't the "main action" of the command anyway.

Thoughts? I think it would allow you to not have to do this custom re-raise on each cmd.